### ISSUE TITLE
[BUG] Dash at line start in math mode breaks mathjax rendering

### ISSUE THREAD

**Issue by fhchl** [shd101wyy/vscode-markdown-preview-enhanced#1795]
**Describe the bug**
Dash at line start in math mode breaks mathjax rendering.
e.g.

```
$$
- 1
- 2
$$
```

Does not render as $-1-2$ anymore, but instead shows some html tags in markdown preview.

**Environment**
Version: 1.82.3
Commit: fdb98833154679dbaa7af67a5a29fe19e55c2b73
Date: 2023-10-02T10:49:50.455Z
Electron: 25.8.1
ElectronBuildId: 24153832
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Linux x64 6.2.0-33-generic

**To Reproduce**
Put 
```
$$
- 1
- 2
$$
```
in md file and press preview.

**Expected behavior**
Do render math correctly

**Screenshots**
![image](https://github.com/shd101wyy/vscode-markdown-preview-enhanced/assets/6132231/ea0cd1a9-68ca-4832-9969-24ed9ed3dac8)

**Comment by fhchl** [comment]
This bug was introduced in the newest version (0.8.0) and does not appear in 0.7.10

**Comment by shd101wyy** [comment]
Yes it's a bug. It seems like we didn't parse the markdown correctly for this case. We considered the content as list items.  

A temp solution for now is to use:

````
```math
- 1
- 2
```
````

I will fix this in the next version :+1:

**Comment by shd101wyy** [comment]
You can also remove the spaces:

```markdown
$$
-1
-2
$$
```

**Comment by shd101wyy** [comment]
Should be fixed now in `0.8.1`. Cheers.

**Comment by fhchl** [comment]
Thanks for the quick fix and the awesome plugin!
